components/ui/TypedWord.vue

```diff
@@ -0,0 +1,113 @@
+<template>
+  <div class="typed-word__container" :class="{ 'typed-word__cursor': showCursor }">
+    <span class="typed-word__content">{{ value }}</span>
+  </div>
+</template>
+
+<script>
+// Implementation based on https://github.com/webnoobcodes/vuejs-typeeffect
+import Vue from 'vue'
+import { Component, Prop } from 'vue-property-decorator'
+
+@Component
+export default class extends Vue {
+  @Prop({ type: String, default: '' }) initialWord
+  @Prop({ type: Array, default: [] }) typeArray
+  @Prop({ type: Number, default: 200 }) typingSpeed
+  @Prop({ type: Number, default: 100 }) erasingSpeed
+  @Prop({ type: Number, default: 3000 }) eraseTextDelay
+  @Prop({ type: Number, default: 3000 }) newTextDelay
+
+  // Initial data
+  value = ''
```

Comment:
Same here:
```suggestion
  value: string = ''
```
We try to leverage TypeScript type inference although it is considered a good practice to annotate declarations and signatures.